lib/rules/function-calc-no-unspaced-operator/__tests__/index.mjs

```diff
@@ -586,60 +599,145 @@ testRule({
+			],
 		},
 		{
-			code: 'a { top: calc(100%/ #{$foo}); }',
+			code: 'a { top: clamp(1px +2px, 3px-4px, none); }',
+			fixed: 'a { top: clamp(1px + 2px, 3px - 4px, none); }',
+			description: 'multiple argument functions',
+			warnings: [
+				{ message: messages.expectedAfter('+'), line: 1, endLine: 1, column: 20, endColumn: 21 },
+				{ message: messages.expectedBefore('-'), line: 1, endLine: 1, column: 29, endColumn: 30 },
+				{ message: messages.expectedAfter('-'), line: 1, endLine: 1, column: 29, endColumn: 30 },
+			],
 		},
 		{
-			code: 'a { top: calc(100% / #{$foo}); }',
+			code: 'a { width: min(25vw+25vw); }',
+			fixed: 'a { width: min(25vw + 25vw); }',
+			description: 'multiple argument functions',
+			warnings: [
+				{ message: messages.expectedBefore('+'), line: 1, endLine: 1, column: 20, endColumn: 21 },
+				{ message: messages.expectedAfter('+'), line: 1, endLine: 1, column: 20, endColumn: 21 },
+			],
 		},
-	],
-
-	reject: [
 		{
-			code: 'a { top: calc(100%- #{$foo}); }',
-			fixed: 'a { top: calc(100% - #{$foo}); }',
-			message: messages.expectedBefore('-'),
-			line: 1,
-			column: 19,
-			endLine: 1,
-			endColumn: 20,
+			code: 'a { padding: calc(1px+ 2px) calc(3px+ 4px); }',
+			fixed: 'a { padding: calc(1px + 2px) calc(3px + 4px); }',
+			description: 'nested math expression',
+			warnings: [
+				{ message: messages.expectedBefore('+'), line: 1, endLine: 1, column: 22, endColumn: 23 },
+				{ message: messages.expectedBefore('+'), line: 1, endLine: 1, column: 37, endColumn: 38 },
+			],
 		},
 		{
-			code: 'a { top: calc(100% -#{$foo}); }',
-			fixed: 'a { top: calc(100% - #{$foo}); }',
-			message: messages.expectedAfter('-'),
-			line: 1,
-			column: 20,
-			endLine: 1,
-			endColumn: 21,
+			code: 'a { top: rgb(from red calc(r+1) calc(g-+2) calc(clamp(10px+2px, g+2, none))); }',
+			fixed:
+				'a { top: rgb(from red calc(r + 1) calc(g- + 2) calc(clamp(10px + 2px, g + 2, none))); }',
+			description: 'nested math expression',
+			warnings: [
+				{ message: messages.expectedBefore('+'), line: 1, endLine: 1, column: 29, endColumn: 30 },
+				{ message: messages.expectedAfter('+'), line: 1, endLine: 1, column: 29, endColumn: 30 },
+				{ message: messages.expectedBefore('+'), line: 1, endLine: 1, column: 40, endColumn: 41 },
+				{ message: messages.expectedAfter('+'), line: 1, endLine: 1, column: 40, endColumn: 41 },
+				{ message: messages.expectedBefore('+'), line: 1, endLine: 1, column: 59, endColumn: 60 },
+				{ message: messages.expectedAfter('+'), line: 1, endLine: 1, column: 59, endColumn: 60 },
+				{ message: messages.expectedBefore('+'), line: 1, endLine: 1, column: 66, endColumn: 67 },
+				{ message: messages.expectedAfter('+'), line: 1, endLine: 1, column: 66, endColumn: 67 },
+			],
 		},
 	],
 });
+
+// testRule({
+// 	ruleName,
+// 	config: [true],
+// 	customSyntax: 'postcss-scss',
+// 	fix: true,
```

Comment:
I ignored this aspect of this rule purely to keep things a bit simpler.

Implementing this is definitely possible.